xpk.py

```diff
@@ -1115,21 +1343,30 @@ def run_gke_cluster_create_command(args) -> int:
               ' the machine type of other cpu nodepools using `--device-type`.')
     machine_type = args.cluster_cpu_machine_type
 
-  # Create the regional cluster with `num-nodes` CPU nodes in the same zone as
-  # TPUs. This has been tested with clusters of 300 VMs. Larger clusters will
-  # benefit from a larger initial `--num-nodes`. After the cluster is created,
-  # the auto-scaler can reduce/increase the nodes based on the load.
   command = (
       'gcloud beta container clusters create'
-      f' {args.cluster} --release-channel rapid --enable-autoscaling'
```

Comment:
Hi Nina,
Are you sure you don't want the default nodepool to be autoscaled as per the incoming workloads?

Reply:
We are short of capacity, we could keep it without autoscaling this time.